backend/api/urls.py

```diff
@@ -1,8 +1,12 @@
-from django.urls import path
+from django.urls import path, re_path
+from django.conf.urls import url
 
+from .views import index, simplePostView
 
-from .views import index
 
 urlpatterns = [
-    path('', index.index, name="index")
+    path('', index.index, name="index"),
+    path(r'service/author/<str:author_id>/posts/', simplePostView.postAuthorPost, name="post-post-view"),
```

Comment:
service should not be part of the path. I believe in the spec if refers to the IP or the domain name.
`http://something.com/author/...`

Reply:
fixed